client/extensions/wp-super-cache/components/navigation/index.jsx

```diff
@@ -53,26 +55,22 @@ const Navigation = ( { activeTab, site, translate } ) => {
 				path = `${ path }/${ tab }`;
 			}
 
-			if ( site ) {
-				path += `/${ site.slug }`;
-			}
-
 			return (
 				<SectionNavTabItem
 					key={ `wp-super-cache-${ tab }` }
-					path={ path }
+					path={ siteSlug && addSiteFragment( path, siteSlug ) }
 					selected={ ( activeTab || Tabs.EASY ) === tab }>
 					{ getLabel( tab ) }
 				</SectionNavTabItem>
 			);
 		} );
 	};
 
-	const pluginUrl = addSiteFragment( '/plugins/wp-super-cache', get( site, 'slug' ) );
+	const pluginPath = '/plugins/wp-super-cache';
 	return (
 		<div>
 			<HeaderCake backText={ translate( 'Plugin Overview' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **88** times:
`translate( 'Overview' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wpsc-broken-navigation-links)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).